examples/ClarifaiPyspark_Example_NB.ipynb

```diff
@@ -0,0 +1,491 @@
+    "Other columns supported in dataframe are - concepts, metadata, geopoints.  \n",
+    "The input column can have file url or path, or can contain raw text.  \n",
+    "If concepts column exists in the dataframe, set labels=True"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "8b81db38-0dab-4b34-8b0e-64241021dca2",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "dataset_obj.upload_dataset_from_table(dataframe=spark_dataframe, input_type='text/image', labels=True/False, csv_type='raw/url/filepath')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "6efdce4d-01c7-4cb5-b595-632c6598469f",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "### e. Upload with custom dataloader\n",
+    "If you have your dataset stored in a different format or need to do any preprocessing, a custom dataloader can be provided.  \n",
+    "You can find examples of different dataloaders [here](https://github.com/Clarifai/examples/tree/main/datasets/upload)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "ac853504-9dcb-4d59-9d30-4b0110e812ba",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "dataset_obj.upload_dataset_from_dataloader(task=\"visual-classification\", split=\"train\", module_dir=\"volume_module_path\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "d7d09fb0-99d5-4f6e-955e-e5d0a02d0335",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "#2. Fetch inputs from dataset in App"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "b7e7f597-6be7-41ff-a20c-624d31bfc509",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "source": [
+    "### a. Get inputs from dataset in your Clarfai App\n",
+    "You can fetch the inputs and its metadata from your dataset in Clarifai App. It returns a list of jsons. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 0,
+   "metadata": {
+    "application/vnd.databricks.v1+cell": {
+     "cellMetadata": {},
+     "inputWidgets": {},
+     "nuid": "d1a6c44c-c8d2-4539-9f62-aad199fa2c39",
+     "showTitle": false,
+     "title": ""
+    }
+   },
+   "outputs": [],
+   "source": [
+    "inputs_response = list(dataset_obj.list_inputs())"
```

Comment:
is `list` still needed, I assume we return as list now

Reply:
We are returning the generator directly from list_inputs() from python sdk